app/app_windows.go

```diff
@@ -73,11 +73,21 @@ func (a *fyneApp) SendNotification(n *fyne.Notification) {
 	title := escapeNotificationString(n.Title)
 	content := escapeNotificationString(n.Content)
 	appID := a.UniqueID()
+	iconFilePath := filepath.Join(os.TempDir(), fmt.Sprintf("fyne-%s.ico", appID))
 	if appID == "" || strings.Index(appID, "missing-id") == 0 {
 		appID = a.Metadata().Name
 	}
 
-	script := fmt.Sprintf(notificationTemplate, title, content, appID)
+	if a.Icon() != nil {
```

Comment:
I think it would be a good idea to also not save the icon each time a notification is sent. You might want to cache the icon insular to Joe we do in https://github.com/fyne-io/fyne/blob/6b2c99e1a3a79e7463a3083350070fc0c583f412/app/app_xdg.go#L122.

Reply:
what do you think about using the rootConfigDir() instead of os.TempDir() to save the icon?

Reply:
Good catch. Maybe name it "icon.png" to be consistent with what the linked code does on Linux?

Reply:
I'm getting ahead of myself and thinking about a feature where the notification icon can be different from the app's default icon. Should we stick with 'notification-icon.png' or is it better to consider this in another PR? It's up to you :)

Reply:
I'd say that we can be consistent with the old code and then implement it everywhere when we add support for using other icons for notifications :)